### PR TITLE
Map access condition.

### LINKED ISSUE #1556: Map accessCondition
See https://github.com/sul-dlss-labs/cocina-descriptive-metadata/blob/master/mods_cocina_mappings/mods_to_cocina_accessCondition.txt

For example, druid:wz014rh6670.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1556

## Why was this change made?
Map access condition.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   80604 (80.604%)
  Different: 18724 (18.724%)
  To Cocina error:     33 (0.033%)
  To Fedora error:     29 (0.029%)
  Missing:     610 (0.61%)
```

After:
```
Status (n=100000):
  Success:   80613 (80.613%)
  Different: 18715 (18.715%)
  To Cocina error:     33 (0.033%)
  To Fedora error:     29 (0.029%)
  Missing:     610 (0.61%)
```

File: app/services/cocina/from_fedora/descriptive/location.rb [hunk 1]
```diff
@@ -24,6 +30,7 @@ def build
             access[:physicalLocation] = physical_locations if physical_locations.present?
             access[:accessContact] = access_contact if access_contact.present?
             access[:url] = url if url.present?
+            access[:note] = note if note.present?
```
Comment (mjgiarlo) [inline app/services/cocina/from_fedora/descriptive/location.rb:33]: not necessary for this PR, but I thought I'd ask: should we toss a `#compact` after the `#tap` so we can ditch the `if` checks? We seem to be moving in that direction generally in DSA mappings.

**Review by mjgiarlo — APPROVED** [review]
LGTM

File: app/services/cocina/from_fedora/descriptive/location.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina/from_fedora/descriptive/location.rb:33]: Yeah, let me make that change.

Comment (justinlittman) [reply app/services/cocina/from_fedora/descriptive/location.rb:33]: Oh, those are empty array, not nils so compact won't work.

**Pull request merged**